projects/kit/components/input-number/input-number.template.html

```diff
@@ -1,6 +1,8 @@
 <tui-primitive-textfield
     #textfield
     tuiValueAccessor
+    tuiTextfieldPrefix=""
```

Comment:
In future we would need to move all prefix/postfix to maskito and deprecate those directives.